lib/registrant.rb

```diff
@@ -0,0 +1,35 @@
+require "./spec/spec_helper"
+
+# The Registrant class represents a single person and their license data.
+# This includes their name, age, and whether they have a driving permit,
+# driver's license, and whether their license has been renewed.
+# Syntax:
+#
+# registrant = Registrant.new({name: "Bill", age: 18})
+#
+# Methods:
+#
+# registrant.earn_permit
+# registrant.permit?
+class Registrant
+  attr_reader :name, :age, :permit
+  attr_accessor :license_data
+
+  def initialize(info)
```

Comment:
This definitely works, however we want to pay attention to the specs and requirements of a project. The interaction pattern asked for a Registrant to be initialized with 2-3 arguments, positional arguments, rather than a hash.